stripper/dz_sirocco.cfg

```diff
@@ -214,7 +261,99 @@ add:
 	"UseRandomTime" "0"
 	"OnTimer" "SpawnSomething*RunScriptCodeforeach(a2,_ in {WeaponMaker=0}) a <- a2;01"
 	"OnTimer" "SpawnSomething*RunScriptCodeforeach(b2,_ in {ForceSpawnAtEntityOrigin=0}) b <- b2;01"
-	"OnTimer" "SpawnSomething*RunScriptCodeEntFire(a+RandomInt(1,22), b, self.GetName(), 0, null);1-1"
+	"OnTimer" "SpawnSomething*RunScriptCodeEntFire(a+RandomInt(1,17), b, self.GetName(), 0, null);1-1"
+}
+
+; Add more prop to container island so player can jump easier without using a bump mine.
+add:
+{
+	"classname" "prop_dynamic_override"
+	"model" "models/props/de_dust/hr_dust/dust_crates/dust_crate_style_02_64x32x64.mdl"
+	"angles" "0 105 0"
+	"renderamt" "255"
+	"rendercolor" "255 255 255"
+	"renderfx" "0"
+	"rendermode" "0"
+	"origin" "-143.474 7710.84 365.093"
+	"skin" "2"
+	"solid" "6"
+    "spawnflags" "0"

```

Comment:
Each of the spawnflags lines here has inconsistent spacing (spaces instead of tabs)